### DISCUSSION TITLE
asp-net-web-api-owin-authenticated-integration-tests-without-authorization-server/

### DISCUSSION THREAD

**Discussion by martijnboland** [martijnboland/blog#52] [Blog comments]
# asp-net-web-api-owin-authenticated-integration-tests-without-authorization-server/

https://blogs.taiga.nl/martijn/posts/asp-net-web-api-owin-authenticated-integration-tests-without-authorization-server/

**Comment by disqus-comments-migrator** [comment]
<em>[gilligan_MH](https://disqus.com/by/gilligan_MH/) commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at maart 11 2016, 04:37</em>

---

<p>nice, but why "async void" though?</p>

**Reply by martijnboland** [reply]
<em>[Martijn Boland](https://disqus.com/by/martijnboland/) commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at maart 11 2016, 05:47, in reply to [gilligan_MH](https://disqus.com/by/gilligan_MH/)'s [comment](https://github.com/martijnboland/blog/discussions/52#discussioncomment-8952984)</em>

---

<p>To be able to await the response and result in the test.</p>
